website/docs/components/dropdown/partials/version-history/4.11.0.md

```diff
@@ -0,0 +1,13 @@
+## 4.11.0
```

Comment:
this could be a bit of a moving target, but right now I would think this would be part of 4.10?

Reply:
I was going to fine-tune it when it hits the main branch.

Reply:
👍